capstone/src/main/java/com/google/sps/servlets/BlobstoreServlet.java

```diff
@@ -19,7 +19,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
nit: could you add a separate directory for this servlet. Also it's a good idea to have a test even if it's really simple. But feel free to do this in a separate change

Reply:
Could I put this file in the same directory as ClubServlet? The Blobstore servlet always directs to the ClubServlet

Reply:
Also, can the test for this servlet be in ClubServletTest.java or should I create a new file?

Reply:
Sure, I think it can be in the same directory as ClubServlet. You should have a separate test though since it's a separate servlet.